### PR TITLE
cassandra: minor documentation fixes

### LINKED ISSUE #2452: [cassandra] documentation minor issues
* Bug Report

**Deviation from expected behavior:**
There are a couple of minor issues with the cassandra docs:
1. In `cassandra.md` there are some extra spaces in front of headers that make it not render correctly on the [website](https://rook.io/docs/rook/v0.9/cassandra.html)
2. In `cassandra-cluster-crd.md` the placement example is wrong.

### PR DESCRIPTION
**Pull request by yanniszark**
**Description of your changes:**
Minor changes in docs.

**Which issue is resolved by this Pull Request:**
Resolves #2452 

**Checklist:**
- [X] Documentation has been updated, if necessary.

[skip ci]

### CODE REVIEW

File: Documentation/cassandra-cluster-crd.md [hunk 1]
```diff
@@ -79,7 +81,6 @@ Rack Settings:
 * `resources`: Defines the CPU and RAM resources for the Cassandra Pods.
 * `placement`: Defines the placement of Cassandra Pods. Has the following subfields:
     * [`nodeAffinity`](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)
-    * [`nodeAntiAffinity`](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)
```
Comment (galexrt) [inline Documentation/cassandra-cluster-crd.md]: I think it would be better to add `nodeAntiAffinity` to the placement type and document it per storage backend if it is available (e.g., nodeAffinity and/or nodeAntiAffinity is not available for Ceph mons). Though this could be okay to add in another PR.

Comment (yanniszark) [reply Documentation/cassandra-cluster-crd.md]: The thing is, nodeAntiAffinity doesn't exist as a field in the Kubernetes API. 
It was a mistake on my part, I thought it existed like podAntiAffinity, but it doesn't.
